docs/3.0rc/develop/task-runners.mdx

```diff
@@ -323,4 +276,52 @@ def sum_it(numbers, static_iterable):
 
 resulting_sum = sum_it([4, 5, 6], [1, 2, 3])
 assert resulting_sum == [10, 11, 12]
+```
```

Comment:
just moved this section, as it seems more advanced than the likely more common need to fetch results from futures